npm run amplify-modelgen giving throw er; // Unhandled 'error' event

Following this [guide ](https://aws.amazon.com/blogs/aws/amplify-datastore-simplify-development-of-offline-apps-with-graphql/) with React Native app, but getting this error 

`$> npm run amplify-modelgen

> test@0.0.1 amplify-modelgen E:\...\amplify\datastore\test
> node amplify\scripts\amplify-modelgen.js

Running codegen...
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: spawn amplify ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
    at onErrorNT (internal/child_process.js:415:16)
    [... lines matching original stack trace ...]
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! test@0.0.1 amplify-modelgen: `node amplify\scripts\amplify-modelgen.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the test@0.0.1 amplify-modelgen script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\...\npm-cache\_logs\2020-01-27T18_53_52_973Z-debug.log`

**Amplify CLI Version**
4.13.1

**To Reproduce**
As per mentioned in doc above. Only change is React Native app.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Node Version :  v10.16.2


**Additional context**

Amplify status is empty (as expected since its a just local Datastore)
$> amplify status

Current Environment: test

| Category | Resource name | Operation | Provider plugin |
| -------- | ------------- | --------- | --------------- |



Detailed logs:
`0 info it worked if it ends with ok
1 verbose cli [ 'E:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Users\\Jitendra\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'amplify-modelgen' ]
2 info using npm@6.7.0
3 info using node@v10.16.2
4 verbose run-script [ 'preamplify-modelgen',
4 verbose run-script   'amplify-modelgen',
4 verbose run-script   'postamplify-modelgen' ]
5 info lifecycle test@0.0.1~preamplify-modelgen: test@0.0.1
6 info lifecycle test@0.0.1~amplify-modelgen: test@0.0.1
7 verbose lifecycle test@0.0.1~amplify-modelgen: unsafe-perm in lifecycle true
8 verbose lifecycle test@0.0.1~amplify-modelgen: PATH: C:\Users\Jitendra\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;E:\xampp\htdocs\git\amplify\datastore\test\node_modules\.bin;C:\Users\Jitendra\bin;E:\Program Files\Git\mingw64\bin;E:\Program Files\Git\usr\local\bin;E:\Program Files\Git\usr\bin;E:\Program Files\Git\usr\bin;E:\Program Files\Git\mingw64\bin;E:\Program Files\Git\usr\bin;C:\Users\Jitendra\bin;C:\Python27;C:\Python27\Scripts;C:\Python38\Scripts;C:\Python38;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\Docker\Docker\Resources\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0;E:\Program Files\TortoiseSVN\bin;E:\Program Files\Git\mingw64\bin;C:\WINDOWS\System32\OpenSSH;E:\Program Files\Microsoft VS Code\bin;C:\Program Files (x86)\Yarn\bin;C:\Program Files\Amazon\AWSCLI\bin;E:\Program Files\nodejs;E:\xampp\php;F:\tools;F:\tools\libwebp-0.4.1-windows-x64\bin;C:\ProgramData\chocolatey\bin;C:\Program Files\Java\jdk1.8.0_231\bin;C:\Users\Jitendra\AppData\Roaming\npm;C:\Users\Jitendra\AppData\Local\Android\Sdk\platform-tools;E:\Program Files\Microsoft VS Code\bin;C:\Users\Jitendra\AppData\Local\Microsoft\WindowsApps;C:\Users\Jitendra\AppData\Local\Yarn\bin;E:\ML\spark-2.3.1-bin-hadoop2.7\bin;C:\Users\Jitendra\AppData\Local\Android\Sdk\emulator;E:\Program Files\Git\usr\bin\vendor_perl;E:\Program Files\Git\usr\bin\core_perl
9 verbose lifecycle test@0.0.1~amplify-modelgen: CWD: E:\xampp\htdocs\git\amplify\datastore\test
10 silly lifecycle test@0.0.1~amplify-modelgen: Args: [ '/d /s /c', 'node amplify\\scripts\\amplify-modelgen.js' ]
11 silly lifecycle test@0.0.1~amplify-modelgen: Returned: code: 1  signal: null
12 info lifecycle test@0.0.1~amplify-modelgen: Failed to exec amplify-modelgen script
13 verbose stack Error: test@0.0.1 amplify-modelgen: `node amplify\scripts\amplify-modelgen.js`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Users\Jitendra\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\index.js:301:16)
13 verbose stack     at EventEmitter.emit (events.js:198:13)
13 verbose stack     at ChildProcess.<anonymous> (C:\Users\Jitendra\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:198:13)
13 verbose stack     at maybeClose (internal/child_process.js:982:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
14 verbose pkgid test@0.0.1
15 verbose cwd E:\xampp\htdocs\git\amplify\datastore\test
16 verbose Windows_NT 10.0.18362
17 verbose argv "E:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Jitendra\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "amplify-modelgen"
18 verbose node v10.16.2
19 verbose npm  v6.7.0
20 error code ELIFECYCLE
21 error errno 1
22 error test@0.0.1 amplify-modelgen: `node amplify\scripts\amplify-modelgen.js`
22 error Exit status 1
23 error Failed at the test@0.0.1 amplify-modelgen script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]`

--- a/packages/amplify-category-auth/src/provider-utils/awscloudformation/question-factories/core-questions.js
+++ b/packages/amplify-category-auth/src/provider-utils/awscloudformation/question-factories/core-questions.js
@@ -1,7 +1,6 @@
 const inquirer = require('inquirer');
 const { uniq, flatten } = require('lodash');
 const chalk = require('chalk');
-const chalkpipe = require('chalk-pipe');
 
 function parseInputs(input, amplify, defaultValuesFilename, stringMapsFilename, currentAnswers, context) {
   // eslint-disable-line max-len
@@ -14,9 +13,8 @@ function parseInputs(input, amplify, defaultValuesFilename, stringMapsFilename, 
   // Can also have some validations here based on the input json
   // Uncool implementation here
 
-  const prefix = input.prefix
-    ? `${'\n'} ${chalkpipe(null, input.prefixColor ? chalk[input.prefixColor] : chalk.green)(input.prefix)} ${'\n'}`
-    : '';
+  const questionChalk = input.prefixColor ? chalk[input.prefixColor] : chalk.green;
+  const prefix = input.prefix ? `${'\n'} ${questionChalk(input.prefix)} ${'\n'}` : '';
 
   let question = {
     name: input.key,

--- a/packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.js
+++ b/packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.js
@@ -1,6 +1,5 @@
 const inquirer = require('inquirer');
 const chalk = require('chalk');
-const chalkpipe = require('chalk-pipe');
 const { uniq, pullAll } = require('lodash');
 const path = require('path');
 const { Sort } = require('enquirer');
@@ -68,7 +67,7 @@ async function serviceWalkthrough(context, defaultValuesFilename, stringMapsFile
     // LEARN MORE BLOCK
     if (new RegExp(/learn/i).test(answer[questionObj.key]) && questionObj.learnMore) {
       const helpText = `\n${questionObj.learnMore.replace(new RegExp('[\\n]', 'g'), '\n\n')}\n\n`;
-      questionObj.prefix = chalkpipe(null, chalk.green)(helpText);
+      questionObj.prefix = chalk.green(helpText);
       // ITERATOR BLOCK
     } else if (
       /*

--- a/packages/amplify-category-interactions/package.json
+++ b/packages/amplify-category-interactions/package.json
@@ -19,7 +19,7 @@
     "fs-extra": "^8.1.0",
     "fuzzy": "^0.1.3",
     "inquirer": "^7.0.3",
-    "inquirer-autocomplete-prompt": "^1.0.1",
+    "inquirer-autocomplete-prompt": "^1.1.0",
     "uuid": "^3.4.0"
   }
 }

--- a/packages/amplify-category-predictions/package.json
+++ b/packages/amplify-category-predictions/package.json
@@ -20,7 +20,6 @@
     "amplify-provider-awscloudformation": "4.26.1",
     "aws-sdk": "^2.608.0",
     "chalk": "^3.0.0",
-    "chalk-pipe": "^3.0.0",
     "fs-extra": "^8.1.0",
     "inquirer": "^7.0.3",
     "open": "^7.0.0",

--- a/packages/amplify-category-predictions/provider-utils/supportedPredictions.js
+++ b/packages/amplify-category-predictions/provider-utils/supportedPredictions.js
@@ -1,5 +1,4 @@
 /* eslint-disable no-multi-str */
-import chalkpipe from 'chalk-pipe';
 import chalk from 'chalk';
 
 const inquirer = require('inquirer');
@@ -42,7 +41,7 @@ Interpret allows you to analyze text for language, entities (places, people), ke
 Infer allows you to perform inference against a cloud endpoint. It’s an advanced feature using Amazon SageMaker, where you have more control over your models.\n\
 Learn More: https://docs.amplify.aws/lib/predictions/intro/q/platform/js';
     helpText = `\n${helpText.replace(new RegExp('[\\n]', 'g'), '\n\n')}\n\n`;
-    questions[0].prefix = chalkpipe(null, chalk.green)(helpText);
+    questions[0].prefix = chalk.green(helpText);
   }
   return questions;
 }

